src/contexts/Graph.js

```diff
@@ -160,10 +186,20 @@ const Graph = ({ children }) => {
       event.txnLink = `https://etherscan.io/tx/${event.txnHash}`
       eventsArray.push(event)
     })
-    graphRinkebyData.data.newReportEntities.forEach((event) => {
+    /*graphRinkebyData.data.newReportEntities.forEach((event) => {
       event.chain = 'Rinkeby Testnet'
       event.txnLink = `https://rinkeby.etherscan.io/tx/${event.txnHash}`
       eventsArray.push(event)
+    })*/
+    graphGoerliData.data.newReportEntities.forEach((event) => {
```

Comment:
Does this `forEach` need to happen twice?

Reply:
Definitely not, think this happened accidentally when I tried to merge...Can I take it out from this review?